webdriver-spec.html

```diff
@@ -3863,7 +3863,15 @@
  <li><p>Return <var>x</var> and <var>y</var> as a pair.
 </ol>
 
-<p>In order to produce a <dfn>pointer-interactable elements tree</dfn>
+<p>An <a>element</a> is <dfn>in view</dfn>
+ if it is a member of its own <a>pointer-interactable paint tree</a>.
+
+<p>An <a>element</a> is <dfn data-lt=obscuring>obscured</dfn>
+ if it is not <a>pointer-interactable</a>
```

Comment:
An element that is below another is not hittable and thus _not_ pointer-interactable. It is the other element _obscuring it_ that is pointer-interactable. This provides a definition for the first, underlying element.